cli/main.c

```diff
@@ -1316,18 +1316,14 @@ static int fw_info(int argc, char **argv)
 	return 0;
 }
 
-static int compare_device_secure_version(struct switchtec_dev *dev, int img_fd,
-				         int assume_yes_ver)
+static int check_device_secure_version(struct switchtec_dev *dev, int img_fd)
```

Comment:
This patch should be squashed... It's making significant changes to a patch in this series and I'd prefer to see the result.

Reply:
Apologies. Had meant to do a rebase and commit --amend. Will update this later.